docs/build/program/apis/data-client.md

```diff
@@ -204,6 +204,66 @@ For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/
 {{% /tab %}}
 {{< /tabs >}}
 
+### TabularDataBySQL
+
+Obtain unified tabular data and metadata, queried with SQL.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `organization_id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The ID of the organization that owns the data.
+- `sql_query` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The SQL query to run.
+
+**Returns:**
+
+- (List[Dict[str, ValueTypes]]): An array of data objects.
+
+```python {class="line-numbers linkable-line-numbers"}
+tabular_data = await data_client.tabular_data_by_sql(
+    organization_id=organization_id,
+    sql_query="SELECT * FROM readings LIMIT 5"
+)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/app/data_client/index.html#viam.app.data_client.DataClient.tabular_data_by_sql).
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### TabularDataByMQL
+
+Obtain unified tabular data and metadata, queried with MQL.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `organization_id` [(str)](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str): The ID of the organization that owns the data.
+- `mql_binary` (List[bytes]): The MQL query to run as a list of BSON documents.
+
+**Returns:**
+
+- (List[Dict[str, ValueTypes]]): An array of data objects.
+
+```python {class="line-numbers linkable-line-numbers"}
+tabular_data = await data_client.tabular_data_by_mql(
+    organization_id=organization_id,
+    mql_query=[b"{ $match: {
+            'robot_id': 'abcdef12-abcd-abcd-abcd-abcdef123456',
+            'component_name': 'my-ultrasonic-sensor',
+            'data.readings.distance': { $gt: .2 } } },
+        { $count: 'numStanding' }"]
```

Comment:
Hey @purplenicole730, I've been trying to figure out how I'd specify the mql query - I know this isn't it but I can't figure out how it would work - can you help?

Reply:
I've never had to actually use mql queries either. I would suggest asking someone from the data team.